config/sql/se/ArtworkTags.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `ArtworkTags` (
+  `ArtworkId` int(11) unsigned NOT NULL,
+  `SubjectTag` varchar(80) NOT NULL,
+  UNIQUE KEY `idxUnique` (`ArtworkId`,`SubjectTag`)
```

Comment:
Tiny suggestion but this unique index might as well be a primary key

```suggestion
  PRIMARY KEY (`ArtworkId`,`SubjectTag`)
```

Reply:
Sure, done.